README.md

```diff
@@ -2,21 +2,22 @@
 
 # Google Earth Engine Plugin for QGIS
 
-Integrates Google Earth Engine with QGIS using the Python API.
+The **QGIS Earth Engine Plugin** integrates [Google Earth Engine](http://earthengine.google.com) with [QGIS](https://qgis.org/) using the [EE Python API](https://github.com/google/earthengine-api/tree/master/python). 
 
-📖 **[User Guide](https://gee-community.github.io/qgis-earthengine-plugin/)**  
-💬 **[Discussions & Support](https://github.com/gee-community/qgis-earthengine-plugin/discussions)**  
-🐞 **[Issue Tracker](https://github.com/gee-community/qgis-earthengine-plugin/issues)**  
+📖 **[User Guide](https://gee-community.github.io/qgis-earthengine-plugin/)**
+💬 **[Discussions & Support](https://github.com/gee-community/qgis-earthengine-plugin/discussions)**
+🐞 **[Issue Tracker](https://github.com/gee-community/qgis-earthengine-plugin/issues)**
 
 ![Add Sentinel-2 image](https://raw.githubusercontent.com/gee-community/qgis-earthengine-plugin/master/media/add_map_layer.png)
 
 ---
 
-## Quickstart Guide
+## 🚀 Quickstart Guide
 
 ### 1️⃣ Install the Plugin
 - Open **QGIS Plugin Manager** (`Plugins > Manage and Install Plugins`).
 - Search for **"Google Earth Engine"** and install it.
+- Restart QGIS after installation.
```

Comment:
We don't need to restart. This shouldn't have been included from my last commit.
```suggestion

```